src/main/webapp/style.css

```diff
@@ -21,6 +21,7 @@ h1 {
 
 h2,
 h3 {
+  background-color: #ececec
```

Comment:
Because this color is already defined in the body, you don't need to redefine it here.